tools/generate_tables_sql_and_testNames.php

```diff
@@ -23,95 +26,118 @@
             continue;
         }
-        switch($bits[0]){
-            //generate the CREATE TABLE syntax
-            case "table":
-                $tablename = $bits[1];
-
-                $filename="../project/tables_sql/".$bits[1].".sql";
-                $output="CREATE TABLE `$bits[1]` (\n";
-                $output.="`CommentID` varchar(255) NOT NULL default '',
-                          `UserID` varchar(255) default NULL,
-                          `Examiner` varchar(255) default NULL,
-                          `Testdate` timestamp DEFAULT CURRENT_TIMESTAMP ON UPDATE CURRENT_TIMESTAMP,
-                          `Data_entry_completion_status` enum('Incomplete','Complete') NOT NULL default 'Incomplete',\n";
+        switch ($bits[0]) {
+        // generate the CREATE TABLE syntax
+        case "table":
+            $tablename = $bits[1];
+            $filename  = "../project/tables_sql/".$bits[1].".sql";
+            $stat      = [ 'Incomplete', 'Complete' ];
+            $output   .= "CREATE TABLE `$bits[1]` (
+    `CommentID` varchar(255) NOT NULL default '',
+    `UserID` varchar(255) default NULL,
+    `Examiner` varchar(255) default NULL,
+    `Testdate` timestamp DEFAULT CURRENT_TIMESTAMP ON UPDATE CURRENT_TIMESTAMP,
+    `Data_entry_completion_status` enum(
+	$stat.join(',')
+     ) NOT NULL default $stat[0], ";
             break;
-            case "page":
-                $pages[] = $bits[2];
-                break;
-
-            //no SQL need be generated.
-            case "title":
-                $title = $bits[1];
-            case "header":
-                break;
+        case "page":
+            $pages[] = $bits[2];
+            continue;
 
-            //generate specific column definitions for specific types of HTML elements
-            default:
-                if($bits[1] == "") {
-                    continue 2;
-                }
-                if($bits[0]=="select"){
-                    $bits[0]=enumizeOptions($bits[3], isset($tablename) ? $tablename : null, $bits[1]);
-                } else if($bits[0]=="selectmultiple"){
-                    $bits[0]="varchar(255)";
-                } else if($bits[0]=="textarea"){
-                    $bits[0]="text";
-                } else if($bits[0]=="text"){
-                    $bits[0]="varchar(255)";
-                } else if($bits[0]=="checkbox") {
-                    $bits[0]="varchar(255)";
-                } else if ($bits[0]=="static") {
-                    $bits[0]="varchar(255)";
-                }
+        // no SQL need to be generated.
+        case "title":
+            $title = $bits[1];
+        case "header":
+            continue;
+            break;
 
-                $bits[2]=htmlspecialchars($bits[2]);
-                $output.="`$bits[1]` $bits[0] default NULL,\n";
+        // generate specific column definitions for specific HTML elements
+        default:
+            if ($bits[1] == "") {
+                continue;
```

Comment:
This will generate a warning. Why are you changing it?

Reply:
@driusan No idea, the change was there when I took over this PR from John.

Reply:
Probably occurred because of a merge conflict. I will fix it.

Reply:
Any reason for [this line](https://github.com/aces/Loris/blob/main/tools/generate_tables_sql_and_testNames.php#L38) to not have been changed?

Reply:
Because that line is outside of the switch statement